cdap-docs/faqs/source/cdap.rst

```diff
@@ -241,6 +232,89 @@ production would have a load balancer in front, which is what you would set
 to a particular router, and if you have both UI and router running on each node, you could
 use ``'127.0.0.1'``.
 
+CDAP services on distributed CDAP aren't starting up due to ``java.lang.ClassNotFoundException``. What should I do?
+-------------------------------------------------------------------------------------------------------------------
+If the CDAP services on a distributed CDAP environment wouldn't start up, you will see errors
+in the logs. You will find in the logs for ``cdap-master`` under ``/var/log/cdap/master*.log``
+errors such as these::
+
+ "Exception in thread "main" java.lang.NoClassDefFoundError:
+   co.cask.cdap.data.runtime.main.MasterServiceMain
+     at gnu.java.lang.MainThread.run(libgcj.so.10)"
+
+Things to check as possible solutions:
+
+1. Check if the JDK being used is :ref:`supported by CDAP <hadoop-install-java-runtime>`::
+
+    java -version
+
+#. Check if the CDAP user is using a :ref:`correct version of the JDK <hadoop-install-java-runtime>`::
+
+    sudo su - <cdap-user> 
+    java -version
+   
+#. Run this command to see if all the CDAP classpaths are included::
+
+    /opt/cdap/master/bin/svc-master classpath | tr ':' '\n'
+   
+   Expect to see (where *<version>* is one of ``0.94``, ``0.96``, or ``0.98``)::
```

Comment:
hbase-compat versions outdated (do we really need to list them?).  currently they are 0.96, 0.98, 1.0, 1.0-cdh, and 1.1